testing/cockroach/force_autoreconf.patch

```diff
@@ -0,0 +1,11 @@
+--- a/Makefile
++++ b/Makefile
+@@ -217,6 +217,8 @@
+ # of them (or their dependencies) change.
+ .bootstrap: $(GITHOOKS) $(GLOCK) GLOCKFILE
+ 	@unset GIT_WORK_TREE; $(GLOCK) sync github.com/cockroachdb/cockroach
++	#Force autoconf to run, otherwise won't work on musl for instance
```

Comment:
From what I can tell, it looks like you're shipping with artifacts generated from an autogen run.  This works well for glibc, but is broken on musl.